### PR TITLE
Remove suggestion to encode secrets online

### PR DESCRIPTION
**Pull request by haakonvt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
 
 ## Inputs
 ### Function metadata in Github Workflow
 #### Required
 1. `function_name`: Name of your function. That name will become an `external_id` and *must* be unique within your project
-2. `function_folder`: Parent folder of function code, defaults to `.` (i.e. the root of your repo)
+2. `function_folder`: Parent folder of function code defaults to `.` (i.e. the root of your repo)
 3. `cdf_deployment_credentials`: Name of Github secrets that holds the API-key that shall be used to deploy the function. 
-That API-key should have following CDF capabilities: `Files:READ`, `Files:WRITE`, `Functions:READ`, `Functions:WRITE`
+That API-key should have the following CDF capabilities: `Files:READ`, `Files:WRITE`, `Functions:READ`, `Functions:WRITE`
```
Comment (pavelzubarev) [inline README.md:10]: Arrrr11!!!

File: README.md [hunk 2]
```diff
@@ -48,17 +48,26 @@ with:
...
+```sh
+$ echo '{"slack-token": "123-my-secret-api-key"}' | python -m base64
+eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo=
+```
+To decode and verify it, do:
+$ echo eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo= | python -m base64 -d
+{"slack-token": "123-my-secret-api-key"}
+```
 Take that string and store it into GitHub secret (COGNITE_FUNCTION_SECRETS from example above, f.ex) 
+Notes: _Keys must be lowercase characters, numbers or dashes (-) and at most 15 characters. You can supply at most 5 secrets_
```
Comment (pavelzubarev) [inline README.md:74]: is it something from Functions? (should we validate that in our code then?)

Comment (haakonvt) [reply README.md:75]: Unsure, the API returns a well-formulated error 🤔

File: README.md (hunk 1 above)
Comment (haakonvt) [reply README.md:10]: The THE-ifying has begun... @scottmelhop

**Review by pavelzubarev — APPROVED** [review]

**Pull request merged**